proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -148,6 +148,7 @@ service UserService {
     option (zitadel.protoc_gen_zitadel.v2.options) = {
       auth_option: {
         permission: "authenticated"
+        org_field: "organization"
```

Comment:
i see that this would map the organization of the request into authz, but it will not be used anyway, right?
and why do we the field in the request at all?

Reply:
My fault, that was a test on the context to maybe check the permission before you query the user, but as it stands now it makes no sense.
Removed it.